nonstandard/QA_scripts/qa_cb.R

```diff
@@ -19,8 +19,8 @@ function_paths <- c(
 source_all(function_paths)
 
 inputs_path <- get_st_data_path()
-results_path <- file.path(get_st_data_path("ST_PROJECT_FOLDER"), "outputs")
-graph_path <- file.path(get_st_data_path("ST_PROJECT_FOLDER"), "outputs", "graphs")
+results_path <- get_st_data_path("ST_PROJECT_FOLDER_OUTPUT")
+graph_path <- file.path(get_st_data_path("ST_PROJECT_FOLDER_OUTPUT"), "graphs")
```

Comment:
These strings are scattered in multiple places. I think it's best to abstract them in a lil helper to make future changes easier

```r
some_name <- function() "ST_PROJECT_FOLDER_OUTPUT"
```

Then, the day we rename this to say "ST_PROJECT_OUTPUT" the diff will show a single change in the body of `some_function()`. This is the idea that DRY code is easy to maintain.

Reply:
on the wishlist